src/components/ProfileList/index.js

```diff
@@ -235,7 +234,13 @@ class ProfileList extends Component {
     let majors = createMajorsListItem(this.props.profile, rowWidths, { gridStyle });
 
     // Creates the Advisors List Item
-    let advisors = createAdvisorsListItem(this.props.profile, rowWidths, { gridStyle });
+    let advisors;
+    // only show on personal profile
+    if(this.props.myProf) {
+      advisors = createAdvisorsListItem(this.props.profile, rowWidths, { gridStyle });
+    } else {
+      advisors = <></>;
```

Comment:
Consider returning null rather than an empty fragment. It is slightly more performant since React doesn't have to create the fragment object.